pkg/kubeapiserver/server/insecure_handler.go

```diff
@@ -28,10 +28,10 @@ import (
 func BuildInsecureHandlerChain(apiHandler http.Handler, c *server.Config) http.Handler {
```

Comment:
if we're trying to match the order of DefaultBuildHandlerChain, we would do this:

```
WithMaxInFlightLimit
WithAudit
WithAuthentication
WithCORS
WithTimeoutForNonLongRunningRequests
WithWaitGroup
WithRequestInfo
WithCacheControl
WithPanicRecovery
```